hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -616,8 +630,8 @@ onOpenNetworkAckSn Environment{party} openState otherParty snapshotSignature sn
                 --         postTx (decrement, v̂, ŝ, η, η𝛼, ηω)
                 & maybePostDecrementTx snapshot multisig
                 -- Spec: if leader(s + 1) = i ∧ T̂ ≠ ∅
-                --         multicast (reqSn, v, ̅S.s + 1, T̂, 𝑈𝛼, txω)
-                & maybeRequestNextSnapshot (number snapshot + 1)
+                -- REVIEW: multicast (reqSn, v, ̅S.s + 1, T̂, S.𝑈𝛼, S.txω)
+                & maybeRequestNextSnapshot snapshot
```

Comment:
why not snapshot + 1 anymore?

Reply:
The `+1` part is now part of `maybeRequestNextSnapshot` as I needed to refactor it by taking the full snapshot (not only the number)